DTS-STN/Alpha-Site
# Removed pages from search indexing

# Description

[#606 Remove signup confirmation/validation pages from search engine indexing](https://trello.com/c/AkxNFyJd)

Added a couple lines to `robots.js` to disallow web crawlers from scraping `confirmation` and `thankyou` pages in production

## Acceptance Criteria

Signup confirmation/validation pages are removed from search engine indexing

## Test Instructions

1. Pull in branch
2. Verify the `robots.js` and its corresponding test file look correct
3. While there's no real way to test if these changes work locally since the newly added rules only apply to production, you can replace the dev rules with the production rules and go to `/robots.txt` to see that it is still functional.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
